### ISSUE TITLE
Notification when exercise cards are available

### ISSUE THREAD

**Issue by avtkal** [tema6120/ForgetMeNot#61]
Could it be possible to get a notification when cards are available to exercise? Is there any way I could do it now or does it have to be implemented? Thank you.

**Comment by tema6120** [comment]
I haven't implemented it so far but I intent to do it. But I have to approach this rationally so that it does not turn into hell, as more and more cards will be added over time. For now, I am considering the following factors that will affect when to show notifications:
    - which decks or list of decks should be counted
    - the minimum number of cards available for the exercise
    - time period when notifications can be shown (for example 17:00 - 22:00)

**Comment by avtkal** [comment]
You are right, I did not think about those factors! It would be great to have it implemented :)
